examples/tutorials/california_house_price.py

```diff
@@ -0,0 +1,197 @@
+from torch_frame.config.text_tokenizer import TextTokenizerConfig
+from torch_frame.data import DataLoader
+from torch_frame.data.mapper import TextEmbeddingTensorMapper
+from torch_frame.datasets import MultimodalTextBenchmark
+from torch_frame.nn import (
+    EmbeddingEncoder,
+    FTTransformer,
+    LinearEncoder,
+    MultiCategoricalEmbeddingEncoder,
+)
+from torch_frame.nn.encoder.stype_encoder import LinearEmbeddingEncoder, LinearModelEncoder, TimestampEncoder
+from torch_frame.typing import NAStrategy, TensorData, TextTokenizationOutputs
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--dataset", type=str, default="california_house_price")
+parser.add_argument("--channels", type=int, default=256)
+parser.add_argument("--num_layers", type=int, default=4)
+parser.add_argument("--batch_size", type=int, default=512)
+parser.add_argument("--lr", type=float, default=0.0001)
+parser.add_argument("--epochs", type=int, default=100)
+parser.add_argument("--seed", type=int, default=0)
+parser.add_argument("--model", type=str,
+                    default="sentence-transformers/all-distilroberta-v1")
+parser.add_argument("--pooling", type=str, default="mean",
+                    choices=["mean", "cls"])
+parser.add_argument("--compile", action="store_true")
+args = parser.parse_args()
+
+torch.manual_seed(args.seed)
+device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+
+class SimpleTextToEmbedding:
+    def __init__(self, model: str, device: torch.device):
+        self.tokenizer = AutoTokenizer.from_pretrained(model)
+        self.model = AutoModel.from_pretrained(model).to(device)
+        self.device = device
+
+    def __call__(self, sentences) -> Tensor:
+        inputs = self.tokenizer(
+            sentences,
+            truncation=True,
+            padding="max_length",
+            return_tensors="pt",
+        )
+        for key in inputs:
+            if isinstance(inputs[key], Tensor):
+                inputs[key] = inputs[key].to(self.device)
+        out = self.model(**inputs)
+        return out.last_hidden_state[:, 0, :].detach().cpu()
+
+
+text_encoder = SimpleTextToEmbedding(model=args.model, device=device)
+
+kwargs = {
+        "text_stype": stype.text_embedded,
+        "col_to_text_embedder_cfg": TextEmbedderConfig(
+            text_embedder=text_encoder,
+            batch_size=5,
+        ),
+    }
+
+# Prepare datasets
+path = osp.join(osp.dirname(osp.realpath(__file__)), "..", "..", "data",
+                args.dataset)
+
+# Load Dataset
+dataset = MultimodalTextBenchmark(root=path, name=args.dataset, num_rows=5000, **kwargs)
+
+model_name = args.model.replace('/', '')
+filename = f"{model_name}_data.pt"
+dataset.materialize(path=osp.join(path, filename))
+dataset.tensor_frame.col_names_dict[stype.embedding] = dataset.tensor_frame.col_names_dict.pop(stype.text_embedded)
+dataset.tensor_frame.feat_dict[stype.embedding] = dataset.tensor_frame.feat_dict.pop(stype.text_embedded)
```

Comment:
These are not needed.

For the tutorial, we assume embeddings are given inside `df`, not something obtained in tensor frame.
I think we can just assume that embedding columns are given in the data frame (although in practice it's not).

Reply:
I am fine with adding some synthetic embedding columns just for illustration purposes in the tutorial html.